Public/Src/FrontEnd/SdkTesting/Helper/TestRunner.cs

```diff
@@ -195,7 +196,7 @@ public bool Run(string testFolder, string specFile, string fullIdentifier, strin
                                     AbsolutePath.Create(frontEndContext.PathTable, testMainFile),
                                 },
                                 CollectionUtilities.EmptyArray<StringId>());
-                            if (!frontEndController.PopulateGraph(cache, graph, engineAbstraction, evaluationFilter, configuration, configuration.Startup))
+                            if (!frontEndController.PopulateGraph(cache, graph, pipGraphFragmentManager, engineAbstraction, evaluationFilter, configuration, configuration.Startup))

```

Comment:
>pipGraphFragmentManager [](start = 80, length = 23)

I don't think this should be a separate argument.

Reply:
done.

---
In reply to: [288666712](https://github.com/ljcollins25/BuildXL/pull/1#discussion_r288666712) [](ancestors = 288666712)